### DISCUSSION TITLE
Active graphs in window

### DISCUSSION THREAD

**Discussion by aviationrn** [TomSchimansky/CustomTkinter#151] [Q&A]
Hello! quick question, is it possible to have active moving graphs using this custom tkinter? We need to get data from another program and have graph it live. Is this possible with the current version?

**Comment by felipetesc** [comment]
Hi. I'm also interested about this, and all related about Python and plots. Can you answer me a question ? Can you point some visual examples for us about what do you want ?
Is it something like this ? 
- https://visme.co/blog/types-of-graphs/
- https://i.stack.imgur.com/vnqOW.jpg
- https://datatofish.com/wp-content/uploads/2018/08/Capture_GUI-1024x361.jpg

Thanks!

**Reply by aviationrn** [reply]
Hello, I'm mostly interested in the 2nd and 3rd link you put in your reply. Only other thing would mainly be pie charts. Mostly just need to integrate custom tkinter with live updatable graphs. Tom below seems to have helped me get on the right track.

**Comment by TomSchimansky** [comment]
You can integrate matplotlib plots like you can do with normal tkinter: https://pythonprogramming.net/how-to-embed-matplotlib-graph-tkinter-gui/

**Reply by aviationrn** [reply]
I understand. I looked at the link you sent and the code makes sense. However, I'm getting an import error listed below. If you by any chance have an idea at least where to find an updated version or the issue it would mean a lot. Thank you for your help and time. https://github.com/spacetelescope/jwst_coronagraph_visibility/issues/20
